yarn-project/aztec.js/src/contract_deployer/deploy_method.ts

```diff
@@ -78,7 +78,7 @@ export class DeployMethod<TContract extends ContractBase = Contract> extends Bas
 
     const contractDeploymentData = new ContractDeploymentData(
       this.publicKey,
-      constructorHash,
```

Comment:
For my mental image, it still made more sense to me that it was the constructor hash actually combining all the things and not just the vk hash. Bu as @iAmMichaelConnor mentioned they he desire to change how contracts are deployed anyway, don't seem worth it to look deeper into that now.

Reply:
> it still made more sense to me that it was the constructor hash actually combining all the things and not just the vk hash.

Maybe I misunderstood but the constructor hash is actually "combining all the things" since part of its preimage is the constructor vk hash (see [this code](https://github.com/AztecProtocol/aztec-packages/blob/1f878be1490e643e1e2ffe097f755268273b73ed/yarn-project/circuits.js/src/contract/contract_deployment_info.ts#L45)).

This change I did didn't break anything because whereever the incorrectly named constructorHash value was used it actually represented the constructorVkHash. [Here](https://github.com/AztecProtocol/aztec-packages/blob/1f878be1490e643e1e2ffe097f755268273b73ed/yarn-project/circuits.js/src/structs/tx_context.ts#L21) is where the value is assigned.